styleguide.config.js

```diff
@@ -81,6 +81,11 @@ module.exports = {
             components:
                 "packages/wonder-blocks-icon-button/components/icon-button.js",
         },
+        {
```

Comment:
🛑 Extra file - also missing the dropdown stuff (although maybe that's intentional?)